x-pack/solutions/security/plugins/security_solution/public/entity_analytics/components/privileged_user_monitoring/components/privileged_user_activity/columns.tsx

```diff
@@ -0,0 +1,304 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import React from 'react';
+import { FormattedMessage } from '@kbn/i18n-react';
+import { capitalize, isArray } from 'lodash/fp';
+import { EuiBadge, EuiButtonIcon, type EuiBasicTableColumn } from '@elastic/eui';
+import type { FlyoutPanelProps } from '@kbn/expandable-flyout';
+import { i18n } from '@kbn/i18n';
+import { getEmptyTagValue } from '../../../../../common/components/empty_value';
+import { PreferenceFormattedDate } from '../../../../../common/components/formatted_date';
+import { UserName } from '../../../user_name';
+import { getRowItemsWithActions } from '../../../../../common/components/tables/helpers';
+import { NetworkDetails } from '../../../network_details';
+import type { TableItemType } from './types';
+import { DocumentDetailsRightPanelKey } from '../../../../../flyout/document_details/shared/constants/panel_keys';
+import { SCOPE_ID } from '../../constants';
+
+const timestampColumn: EuiBasicTableColumn<TableItemType> = {
+  field: '@timestamp',
+  truncateText: true,
+  name: (
+    <FormattedMessage
+      id="xpack.securitySolution.entityAnalytics.privilegedUserMonitoring.userActivity.columns.timestamp"
+      defaultMessage="Timestamp"
+    />
+  ),
+  width: '20%',
```

Comment:
An idea, maybe a bad one,  could we declare a const object with all of the column width percentage values together so that its easy to see the differences/tweak them? 

const COLUMN_WIDTHS = { TIMESTAMP: '20%', ...